Gopkg.lock

```diff
@@ -56,6 +56,14 @@
   pruneopts = ""
   revision = "09c41003ee1d5015b75f331e52215512e7145b8d"
 
+[[projects]]
```

Comment:
anyway to avoid adding deps please?

Reply:
Sure! I'm about to head home for the day, but will likely peek back at this tomorrow. Thanks for the comment.